File: Conditionals/Time.c
// Eve Richardson, Time of day C
#include <stdio.h>
#include <time.h>

int main(void){
    time_t seconds;
    seconds = time(NULL);
    printf("Seconds since start of 1970 = %d\n", seconds);

    time_t rawtime;
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("Current time and date it %s\n", asctime(timeinfo));

    //current hour
    time_t now = time(NULL);
    struct tm *tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);

    return 0;
}

File: Conditionals/Time.py
#Eve Richardson Time of day Python

import time

#print(time.gmtime(0)) 
#getting the very first instance of time

current = time.time()
#time in seconds since gmtime

#current time as we are used to seeing it
now = time.ctime(current)
print(now)

#get just the hour
local_time = time.localtime(current)
print(local_time)
minutes = local_time.tm_min
print(minutes)
hours = local_time.tm_hour
print(hours)